### ISSUE TITLE
Reproduce experimental results using pre-trained models

### ISSUE THREAD

**Issue by KotaShimomura** [avishkarsaha/translating-images-into-maps#41]
I ran the pre-trained model on the nuScenes mini dataset.

However, the model's predicted results appear to be collapsing.
Does anyone have the same phenomenon or the correct mask predicted?

I am using checkpoint-0020.pht.gz.
Fine-tuning is not done.
![val_output_epoch1_iter0](https://github.com/avishkarsaha/translating-images-into-maps/assets/65890986/dbb6fba7-d866-4dda-87bd-8f9d882fb2ae)
